gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/cloud/storage/contrib/nio/it/ITGcsNio.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
My comment was about not putting a date that is prior to when the file was created (this was created in 2016 if I am not mistaken). Keeping (or adding) 2015 to files created in 2015 should (I am not an expert) be fine.